controllers/common.go

```diff
@@ -550,3 +551,71 @@ func GetCommonRbacRules(privileged bool) []rbacv1.PolicyRule {
 
 	return []rbacv1.PolicyRule{rbacPolicyRule}
 }
+
+// Some frameworks like (e.g., Tobiko and Horizon) require password value to be
+// present in clouds.yaml. This code ensures that we set a default value of
+// 12345678 when password value is missing in the clouds.yaml
+func EnsureCloudsConfigMapExists(
+	ctx context.Context,
+	instance client.Object,
+	helper *helper.Helper,
+	labels map[string]string,
+) (ctrl.Result, error) {
+	const openstackConfigMapName = "openstack-config"
+	const testOperatorCloudsConfigMapName = "test-operator-clouds-config"
+
+	cm, _, _ := configmap.GetConfigMap(
+		ctx,
+		helper,
+		instance,
+		openstackConfigMapName,
+		time.Second*10,
+	)
+	if cm.Name == testOperatorCloudsConfigMapName {
+		return ctrl.Result{}, nil
+	}
+
+	cm, _, _ = configmap.GetConfigMap(
```

Comment:
I am not sure why this part is present 2 times in the function. I don't know if it's a copy-paste mistake or is needed there.

Reply:
I don't see this fixed.

Reply:
The function should be called two times. The issue was that we called it with `openstackConfigMapName` in both function calls.